src/runtime/hvx_128.ll

```diff
@@ -237,12 +237,31 @@ define weak_odr <128 x i8> @halide.hexagon.shl.vub.ub(<128 x i8> %a, i8 %b) noun
   ret <128 x i8> %r
 }
 
+define weak_odr <128 x i8> @halide.hexagon.shl.vub.b(<128 x i8> %a, i8 %b) nounwind uwtable readnone alwaysinline {
+  %a_32 = bitcast <128 x i8> %a to <32 x i32>
+  %bw = zext i8 %b to i32
```

Comment:
`sext` (sign extend) instead of `zext` (zero extend)

Reply:
Done